.github/FUNDING.yml

```diff
@@ -1,4 +1,4 @@
 # These are supported funding model platforms
-custom: ["https://paypal.me/anodev",
+custom: ["https://paypal.me/anodevru",
```

Comment:
@OPHoperHPO I cannot donate via paypal.
It says something like "The receiver has reached his receive-limit and must provide additional information to remove the limit"

Reply:
> @OPHoperHPO I cannot donate via paypal.
> It says something like "The receiver has reached his receive-limit and must provide additional information to remove the limit"
> 
> EDIT: ok it works when I send less :)

Hi @pubkey,

The first thing I have to say is thanks for the financial support, it really helps to support and develop the project further! 

Now about the problem you have with PayPal. 
Sorry, PayPal has just suspended my account temporarily. I seem to have solved the problem, but I'm not sure about that yet.